Makefile

```diff
@@ -1,34 +1,92 @@
+.POSIX:
+all:
+
+# Recipes
+
+## Build ameba
+##   $ make
+## Run tests
+##   $ make test
+## Install ameba
+##   $ sudo make install
```

Comment:
This list IMO should be either completed to match the targets 1:1, or dropped altogether.

Reply:
The idea of the recipe section is to highlight the most common commands and show some examples for how to use them. I suppose the options are very straightforward, so it might not be super necessary (compared to `make help` of other projects like [Crystal](https://github.com/crystal-lang/crystal/blob/master/Makefile) and [Shards](https://github.com/crystal-lang/shards/blob/master/Makefile)).
I think it's nevertheless helpful to have some recipes to start with.
But it certainly doesn't make sense to list *all* targets here.

Reply:
Makes sense, thanks :)